src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double longerSIde;
+    private final double height;
+
+    public Rectangle(double longerSIde, double height, String color) {
+        super(color);
+        this.longerSIde = longerSIde;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return longerSIde * height;
+    }
+
+    @Override
+    public void toDraw() {
```

Comment:
The method name 'toDraw' does not follow Java naming conventions for methods. It should be a verb, such as 'draw'. Also, ensure that it matches the method name defined in the 'Drawable' interface.